core/actions/actions.go

```diff
@@ -5,26 +5,41 @@
 package actions
 
 import (
+	"strings"
+
 	"github.com/juju/charm/v9"
 )
 
-// JujuRunActionName defines the action name used by juju-run.
-const JujuRunActionName = "juju-run"
+// JujuExecActionName defines the action name used by juju-exec.
+const JujuExecActionName = "juju-exec"
+
```

Comment:
probably
```go
const JujuExecActionNameLegacy = "juju-run"
```

then use it for those legacy checks, like `IsJujuExecAction`, `HasJujuExecAction`, etc.